Doc/whatsnew/3.14.rst

```diff
@@ -61,12 +65,26 @@ Summary -- release highlights
 .. This section singles out the most important changes in Python 3.14.
    Brevity is key.
 
+Python 3.14 beta is the pre-release of the next version of the Python
+programming language, with a mix of changes to the language, the
+implementation and the standard library.
+
+The biggest changes to the implementation include template strings (:pep:`750`),
+deferred evaluation of annotations (:pep:`649`),
+and a new type of interpreter that uses tail calls.
+
+The library changes include the addition of a new :mod:`!annotationlib` module
+for introspecting and wrapping annotations (:pep:`649`),
+a new :mod:`!compression.zstd` module for Zstandard support (:pep:`784`),
```

Comment:
Using `:mod:` when the module doesn't exist doesn't really delineate the module name as I'd expect, at least on my browser and in dark mode
![image](https://github.com/user-attachments/assets/f7f136a8-98e9-4ad2-872e-578a15c64447)

In https://github.com/python/cpython/pull/133495 I used ``module\`\`, which looks a bit better IMO?
![image](https://github.com/user-attachments/assets/78ca116c-253c-4f63-9627-2fdd59b25157)

Regardless, we should probably use the same syntax in both places.

Reply:
Ah, you just replied there. Ignore this!